src/test/java/CheckboxesTest.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class CheckboxesTest extends BaseTest {
+    private List<WebElement> elementsOfCheckbox;
+
+    @BeforeClass
+    public void getCheckboxElements() {
+        driver.get("http://the-internet.herokuapp.com/checkboxes");
+        elementsOfCheckbox = driver.findElements(By.cssSelector("[type = checkbox]"));
+    }
+
+    @Test
+    public void checkCheckbox1() {
+        WebElement checkBox1 = elementsOfCheckbox.get(0);
```

Comment:
it is better to check elementsOfCheckbox size before using .get(0);

Reply:
I've fixed it